tests/ssl-opt.sh

```diff
@@ -1893,6 +1910,34 @@ run_test    "Authentication: client no cert, ssl3" \
             -C "! mbedtls_ssl_handshake returned" \
             -S "X509 - Certificate verification failed"
 
+# Tests for CA list suppression
+
+run_test    "Authentication: CA suppression disabled (default)" \
```

Comment:
Please update the test descriptions (e.g. “Authentication: send CA list in CertificateRequest (default)”, “Authentication: suppress CA list in CertificateRequest” or “Authentication: do not send CA lis in CertificateRequest”).